### ISSUE TITLE
server panic raised when mqtt broker is disconnected.

### ISSUE THREAD

**Issue by chespinoza** [thingful/device-hub#92]
I've been having this issue every day as I'm loading and unloading resources config continually.

```
Christians-MBP:darwin-amd64 christian[client-config-file]$ ./device-hub server
2017/07/21 09:24:45 Listening on: [:50051]
2017/07/21 10:35:15 mqtt broker connected
2017/07/21 10:54:00 mqtt broker disconnectedpingresp not received, disconnecting
panic: mqtt broker disconnectedpingresp not received, disconnecting

goroutine 370 [running]:
log.Panic(0xc42034bf90, 0x2, 0x2)
	/usr/local/go/src/log/log.go:322 +0xc0
github.com/thingful/device-hub/listener.Register.func1.1(0x1a527a0, 0xc420270000, 0x1a45340, 0xc4200121f0)
	/Users/christian/go/src/github.com/thingful/device-hub/listener/listeners.go:71 +0xd2
created by github.com/thingful/device-hub/vendor/github.com/eclipse/paho%2emqtt%2egolang.(*client).internalConnLost
	/Users/christian/go/src/github.com/thingful/device-hub/vendor/github.com/eclipse/paho.mqtt.golang/client.go:418 +0x109
```

**Comment by chespinoza** [comment]
The panic is deliberately raised due issue #27, it's happening after unload mqtt resources from server.

**Comment by chespinoza** [comment]
Addressed in #93
